package.json

```diff
@@ -27,11 +27,13 @@
     "inference"
   ],
   "scripts": {
+    "fix:format": "prettier --write \"src/**/*.ts\"",
+    "fix:lint": "eslint --fix --ext .ts ./src",
+    "fix": "yarn fix:lint && yarn fix:format",
```

Comment:
@colinhacks please let me know, whether you agree with:
- organizing sub scripts with a `:` separator

Reply:
Works for me!